python/requirements/ml/requirements_no_deps.txt

```diff
@@ -3,4 +3,4 @@
 # (without constraints), but their dependencies may be overwritten afterwards.
 
 # mosaicml requires importlib-metadata>5, but flake8 is not compatible with it
-mosaicml==0.12.1
+mosaicml==0.12.1
```

Comment:
what exactly changed here?

Reply:
supposedly no change, let me see if i can get rid of this